### DISCUSSION TITLE
Would ListToCsvStringDeserializer be a good candidate to move into kiwi?

### DISCUSSION THREAD

**Discussion by sleberknight** [kiwiproject/service-discovery-client#142] [Q&A] [answered]
Does it make any sense to migrate `ListToCsvStringDeserializer` to the base kiwi project? Or is it way too specific that nothing (other than this project) is likely to use it?

The use case in this project is for a CSV list of Eureka URLs, so for example in a YAML config we can have _either:_

```yaml
registryUrls: eureka-url-1,eureka-url-2
```
_or_

```yaml
registryUrls:
  - eureka-url-1
  - eureka-url-2
```

and either way will work.

Making this generic and moving into kiwi would then allow for any similar configuration in which you want to be able to specify either a CSV string (as in the first example), or a list of items (like in the second example).

**Comment by chrisrohr** [comment, marked as answer]
Sure why not :-) :beers:

**Reply by sleberknight** [reply]
Created issue https://github.com/kiwiproject/kiwi/issues/608 in kiwi. This discussion is now effectively closed.
